examples/swift/async_await/ContentView.swift

```diff
@@ -0,0 +1,116 @@
+import Envoy
+import Foundation
+import SwiftUI
+
+// MARK: - Content View
+
+private struct ContentRow: Identifiable {
```

Comment:
This file could easily be split up, but is more helpful as an example when you can read the whole thing from top to bottom. Given that this fits in 116 lines, I think that's the right tradeoff.